service/models/User.js

```diff
@@ -32,7 +32,8 @@ const UserSchema = new mongoose.Schema({
   },
   password: { type: String, required: true },
   decks: [DeckSchema],
-  active: { type: Boolean, default: true }
+  active: { type: Boolean, default: true },
+  role: {type: ['user', 'superuser', 'admin'], default: 'user'}
```

Comment:
Please use the suggested code from this slack message https://nscad.slack.com/archives/C02RK1XATNK/p1646859988936869